src/navigation/index.tsx

```
@@ -11,211 +12,260 @@ import { Settings } from './screens/Settings';
-          <Text style={{
-            ...typography.caption,
-            fontFamily: 'Outfit',
-            color: color,
-            fontSize: 12,
-            marginTop: 0,
-          }}>
-            Home
-          </Text>
-        ),
+// Custom navigation component that uses the theme
+export function Navigation(props: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)